app/[locale]/what-is-ether/page.tsx

```diff
@@ -0,0 +1,758 @@
+              <h2 id={getId(tocItems[2].url)} className="scroll-mt-28">
+                {tocItems[2].title}
+              </h2>
+              <p>
+                {t.rich(
+                  "page-what-is-ether-how-to-send-and-receive-eth-description-1",
+                  {
+                    strong: Strong,
+                  }
+                )}
+              </p>
+              <p>
+                {t.rich(
+                  "page-what-is-ether-how-to-send-and-receive-eth-description-2",
+                  {
+                    strong: Strong,
+                  }
+                )}
+              </p>
+              <p>
+                {t.rich(
+                  "page-what-is-ether-how-to-send-and-receive-eth-description-3",
+                  {
+                    strong: Strong,
+                    howToUseWallet: (chunks) => (
+                      <Link href="/guides/how-to-use-a-wallet/">{chunks}</Link>
+                    ),
+                  }
+                )}
+              </p>
+              <Alert variant="warning">
+                <AlertEmoji text="💡" />
+                <AlertContent>
+                  <p>
+                    {t.rich("page-what-is-ether-how-to-buy-eth-description-3", {
+                      strong: Strong,
+                    })}
+                  </p>
+                  <p className="mt-2">
+                    {t.rich(
+                      "page-what-is-ether-how-to-send-and-receive-eth-description-4",
+                      {
+                        strong: Strong,
+                      }
+                    )}
+                  </p>
+                  <UnorderedList className="mb-0 mt-2 [&>li]:mb-0.5">
+                    <ListItem>
+                      {t.rich(
+                        "page-what-is-ether-how-to-send-and-receive-eth-description-5",
+                        {
+                          strong: Strong,
+                        }
+                      )}
+                    </ListItem>
+                    <ListItem>
+                      {t.rich(
+                        "page-what-is-ether-how-to-send-and-receive-eth-description-6",
+                        {
+                          strong: Strong,
+                        }
+                      )}
+                    </ListItem>
+                  </UnorderedList>
+                </AlertContent>
+              </Alert>
+              <p>
+                {t(
+                  "page-what-is-ether-how-to-send-and-receive-eth-description-7"
+                )}
+              </p>
+              <LinkWithArrow href="/what-is-ethereum/">
+                {t(
+                  "page-what-is-ether-how-to-send-and-receive-eth-description-8"
+                )}
+              </LinkWithArrow>
+            </div>
+          </Section>
+
+          <Section
+            id={getId(tocItems[3].url)}
+            className="-scroll-mt-80 space-y-14"
+          >
+            <div className="space-y-6">
```

Comment:
Just noting this nested div is a bit redundant since it's by itself inside Section... can just update to use `space-y-6` and remove the nested div.. and similar to above, should be able to remove `-scroll-mt-80` here (and throughout)